test/test-lib/supertest.ts

```diff
@@ -3,9 +3,9 @@ import * as _ from 'lodash';
 import * as supertest from 'supertest';
 import { User } from '../../src/platform/jwt';
 
-type UserObjectParam = Partial<User & { token: string }>;
+export type SupertestUser = string | Partial<User & { token: string }>;
```

Comment:
I'd rather break it down to `ApiKey | Token` so test code can specify just one if it wants, with token being `{ token: string } & AnyObject`. It's also more accurate to what the code actually does